src/tests/ftest/cart/ctl/cart_ctl_one_node.yaml

```diff
@@ -26,17 +21,12 @@ hosts: !mux
     config: one_node
     test_servers:
       - boro-A
-    test_clients_1:
+    test_clients:
```

Comment:
Why multiple boro-As here for client?

Reply:
Ah, I thought each test binary got its own hostname.  Fixed: 41e59630884bd6df1c51b952551ec61140c52e3c.